src/styles/globalStyles.scss

```diff
@@ -34,8 +34,12 @@ body {
 
 img {
 	-webkit-user-drag: none;
-	-webkit-user-select: none;
+	user-select: none;
 }
 a {
 	text-decoration: none;
```

Comment:
why none?

Reply:
To prevent highlighting on dragging oar CTRL+A

Reply:
if i wanted to contribute to your project i will expect to a tag to have a default behavior not a text decoration none